proto/config.proto

```diff
@@ -95,4 +95,7 @@ message FrontendConfig {
 
   // Whether to show encryption settings.
   bool customer_managed_encryption_keys_enabled = 31;
+
+  // True if multiple suggestions providers are configured.
+  bool multiple_suggestion_providers = 32;
```

Comment:
Might make sense to return a list of strings for this so it's a little more pluggable, WDYT?

Reply:
I thought about it, but right now I'm displaying a big warning message specific to OpenAI - so sending all of the fields needed via this frontend config seemed like overkill.

I promise I'll do that if we add a third provider :)